docs/contributor/releasing.md

```diff
@@ -1,41 +1,40 @@
 # Release Process
 
-1. For new major and minor versions: Create a release branch in the `telemetry-manager` repository. The name of this branch must follow the `release-x.y` pattern, such as `release-1.0`.
-
-
-   ```bash
+This release process covers the steps to release new major and minor versions for the Kyma Telemetry Component.
+
+1. Bump the `telemetry-manager/main` branch with the new versions for the dependent images.
+   Create a PR to `telemetry-manager/main` with the following changes:
```

Comment:
Considering you specified the repo in the following branch, why not here as well?

Reply:
disregard this; of course it's written here.